Foundation/Thread.swift

```diff
@@ -95,6 +107,9 @@ open class Thread : NSObject {
     }
 
     open class func sleep(until date: Date) {
+#if os(Windows)
+        NSUnimplemented()
```

Comment:
doesnt windows have `usleep` in the terms of this: https://docs.microsoft.com/en-us/windows/desktop/api/synchapi/nf-synchapi-sleep?

Reply:
Yeah, these should be implemented using `Sleep`.  I just couldn't figure out how to make this precise.

Reply:
I don’t think it needs to be nanosecond precise. So the time interval *1000 should work just fine imho